scripts/bootstrap.js

```diff
@@ -21,4 +21,13 @@ const ciCommand = npmCmd === 'ci'
 const commandArgs = ciCommand ? '--ci' : '--no-ci'
 const environmentArgs = ciEnvironment ? '--concurrency 1 --loglevel debug' : ''
 const cmd = `npm run lerna -- bootstrap ${commandArgs} ${environmentArgs}`
+
 childProc.execSync(cmd, {stdio: 'inherit'})
+
+// Symlink local dependencies
+// A bug manifests itself on Windows where lerna bootstrap fails to generate the `pwa-kit-cli` bin shims
+// in the bin file. As a result, Windows runs the `pwa-kit-cli` bin file using Windows Host Script instead
+// of Node. We run the lerna link command to fix the shims in the `pwa-kit-cli` bin file on Windows.
+if (process.platform === 'win32') {
+  childProc.execSync('npm run lerna link --force-local', {stdio: 'inherit'})
```

Comment:
The workaround works now, but now we see warnings after running `npm ci` on the monorepo on Windows:

```
lerna notice cli v3.22.1
lerna info Symlinking packages and binaries
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-runtime dependency of pwa-kit-dev, but links to different location. Replacing with updated symlink...
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-runtime dependency of pwa-kit-react-sdk, but links to different location. Replacing with updated symlink...
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-dev dependency of template-express-minimal, but links to different location. Replacing with updated symlink...
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-runtime dependency of template-express-minimal, but links to different location. Replacing with updated symlink...
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-dev dependency of retail-react-app, but links to different location. Replacing with updated symlink...
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-react-sdk dependency of retail-react-app, but links to different location. Replacing with updated symlink...
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-runtime dependency of retail-react-app, but links to different location. Replacing with updated symlink...
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-react-sdk dependency of typescript-minimal, but links to different location. Replacing with updated symlink...
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-dev dependency of typescript-minimal, but links to different location. Replacing with updated symlink...
lerna WARN EREPLACE_OTHER Symlink already exists for pwa-kit-runtime dependency of typescript-minimal, but links to different location. Replacing with updated symlink...
added 811 packages in 87.194s
```

Reply:
Actually, it is the warnings from running `lerna link`

Reply:
Yes, the warnings come from the workaround, indicating that something is not right. 
The workaround is good enough to ship it, but I suggest creating a new ticket to investigate a better solution.